packages/unit/package.json

```diff
@@ -61,13 +65,22 @@
   },
   "files": [
     "dist",
-    "README.md"
+    "README.md",
+    "*.d.ts"
   ],
   "dependencies": {
     "@suites/core.unit": "^3.0.0",
     "@suites/types.common": "^3.0.0",
     "@suites/types.di": "^3.0.0",
-    "@suites/types.doubles": "^3.0.0"
+    "@suites/types.doubles": "^3.0.0",
+    "@suites/doubles.jest": "^3.0.0",
+    "@suites/doubles.vitest": "^3.0.0",
+    "@suites/doubles.sinon": "^3.0.0"
```

Comment:
Adding those types here seems problematic since the `unit` package isn't linked to any dependency injection or mocking library adapter (e.g. `@suites/types.lib`). Is there a specific reason for including them?